mdata/chunk/sizes.go

```diff
@@ -0,0 +1,60 @@
+package chunk
+
+import (
+	"sync"
+)
+
+type Size uint8
+
+var ChunkSizes = map[Size]uint32{
+	1:  1,
+	2:  5,
+	3:  10,
+	4:  15,
+	5:  20,
+	6:  30,
+	7:  60,    // 1m
+	8:  90,    // 1.5m
+	9:  120,   // 2m
+	10: 180,   // 3m
+	11: 300,   // 5m
+	12: 600,   // 10m
+	13: 900,   // 15m
+	14: 1200,  // 20m
+	15: 1800,  // 30m
+	16: 2700,  // 45m
+	17: 3600,  // 1h
+	18: 5400,  // 1.5h
+	19: 7200,  // 2h
+	20: 9000,  // 2.5h
+	21: 10800, // 3h
+	22: 14400, // 4h
+	23: 18000, // 5h
+	24: 21600, // 6h
+	25: 25200, // 7h
+	26: 28800, // 8h
+	27: 32400, // 9h
+	28: 36000, // 10h
+	29: 43200, // 12h
+	30: 54000, // 15h
+	31: 64800, // 18h
+	32: 86400, // 24h
+}
+
+var RevChunkSizes = make(map[uint32]Size, len(ChunkSizes))
+
+var initMutex sync.Mutex
+var initialized = false
```

Comment:
At the time of writing this part I looked up the docs, but they were not explicit about what happens if there are multiple threads importing this package: https://golang.org/doc/effective_go.html#init

If you're sure about that I'll remove the lock